### PR TITLE
Develop jc

### PR DESCRIPTION
**Pull request by onlimii**

### CODE REVIEW

File: lib/akamai_rest_ccu.rb [hunk 1]
```diff
@@ -1,44 +1,61 @@
...
+      )
+
+      post_request.body = if type == :url
+                            {
+                              'hostname' => opts[:url],
+                              'objects' => opts[:objects] || ['/']
+                            }.to_json
+                          elsif type == :cpcode
+                            {'objects' => opts[:objects]}.to_json
+                          end
```
Comment (li-thy-um) [inline lib/akamai_rest_ccu.rb]: what if a invalid type ?

Comment (onlimii) [inline lib/akamai_rest_ccu.rb]: body will just become nil and call will be failed with a 404

**Comment by ming-relax** [comment]
@onlimii you may want to make this repo private :)

**Comment by onlimii** [comment]
@ming-relax lol, should be fine. They are gonna release this API soon and users still need to turn on the option to benefit this.

**Pull request merged**